### DISCUSSION TITLE
log file

### DISCUSSION THREAD

**Discussion by amicwall** [Ride-The-Lightning/c-lightning-REST#122] [Q&A]
I can't find the log file: global.logger.log

**Comment by ShahanaFarooqui** [comment]
Please refer to your configuration to find the logger being used by you.

**Reply by amicwall** [reply]
which configuration, this ?

{
    "PORT": 3001,
    "DOCPORT": 4001,
    "PROTOCOL": "https",
    "EXECMODE": "production",
    "RPCCOMMANDS": ["*"],
    "DOMAIN": "localhost"
}
